test/functional/test-action.js

```diff
@@ -969,6 +966,31 @@ describe('Core events', () => {
     const element = {tagName: 'target2', nodeType: 1};
     const event = {target: element};
     handler(event);
-    expect(action.trigger).to.have.been.calledWith(element, 'change', event);
+    expect(action.trigger).to.have.been.calledWith(
+        element,
+        'change',
+        sinon.match.any);
```

Comment:
The custom event doesn't have the target on it, but the target is still passed through as the first argument to `trigger`